packages/walletconnect-v1/src/types.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference path='../../../node_modules/@walletconnect/types-v1/index.d.ts' />
```

Comment:
Yeah I also had problems with the v1 type imports. I think it's an issue with the lib itself. I ended up just copying some types over but this is a clever workaround